www/src/pages/docs/index.js

```diff
@@ -22,6 +22,10 @@ class IndexRoute extends React.Component {
               Get started
             </h1>
             <p>Gatsby is a blazing fast modern site generator for React.</p>
+      <p> This quick start is intended for intermediate to advanced developers. For a gentler intro to Gatsby, <Link to="/tutorial/">head to our tutorial!</Link> </p>
+           <h2>Prerequisites</h2>
+             <p> <strong> Node.js</strong>: Before installing Gatsby, make sure you have Node.js and npm package manager installed on your system. If not, go to <a href="https://nodejs.org/" target="_blank">nodejs.org</a> and install a recent version for your operating system. </p>
```

Comment:
@surjithctly nice ! Can you also add `rel="noopener"` to the link on this line as well?

Reply:
Yes. Done 👍